README.md

```diff
@@ -1,19 +1,26 @@
-# Description
+# TONsdk
+[![PyPI](https://img.shields.io/pypi/v/tonsdk?color=blue)](https://pypi.org/project/tonsdk/)
+[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/tonsdk)](https://pypi.org/project/tonsdk/)
+[![Downloads](https://static.pepy.tech/badge/tonsdk)](https://pepy.tech/project/tonsdk)

```

Comment:
But this part is very good. We should move it to examples' README and merge everything into master